src/main/java/io/github/apace100/apoli/power/ActionOverTimePower.java

```diff
@@ -21,6 +22,9 @@
     private final Consumer<Entity> fallingAction;
 
     private boolean wasActive = false;
+    private boolean initialTicksSet = false;
+
+    private int initialTicks;
```

Comment:
Use type `Integer` instead of `int` as it allows a null value.
```suggestion
    private Integer initialTicks;
```